daemon/openrazer_daemon/misc/battery_notifier.py

```diff
@@ -28,6 +27,7 @@ def __init__(self, parent, device_id, device_name):
         self._notify2 = notify2 is not None
 
         self.event = threading.Event()
+        self.frequency = 0
```

Comment:
Can we keep the default at a sane value like `60 * 10` from before? Just in case a device forgets to set the frequency.

Reply:
looks like they are already defaulting to 60*10 when they are reading the config value?

Reply:
> looks like they are already defaulting to 60*10 when they are reading the config value?
correct.
Should I add a comment saying that the variable is assigned to elsewhere?

Reply:
Would love to get this feature in! Right now i went in and manually changed the code after i installed to change the notification frequency 🤣 how does this look @z3ntu ?